File: ios/WrapperOld.m
#import <React/RCTBridgeModule.h>
#import <NamiApple/NamiApple.h>

@interface RCT_EXTERN_MODULE(WrapperOld, NSObject)

RCT_EXTERN_METHOD(multiply:(float)a withB:(float)b
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

+ (BOOL)requiresMainQueueSetup
{
  return NO;
}

@end

File: ios/WrapperOld.swift
import ObjectiveC
import Foundation
//import libxcode
//import NamiApple
import libxcode

@objc(WrapperOld)
class WrapperOld: NSObject {

  @objc(multiply:withB:withResolver:withRejecter:)
  func multiply(a: Float, b: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
//      let instance = multiplyLib()

    resolve(a*b)
  }
}
